llarp/config/config.cpp

```diff
@@ -852,6 +871,46 @@ namespace llarp
             "Permanently map a `.loki` address to an IP owned by the snapp. Example:",
             "mapaddr=whatever.loki:10.0.10.10 # maps `whatever.loki` to `10.0.10.10`.",
         });
+    // extra [network] options
+    // TODO: probably better to create an [exit] section and only allow it for routers
```

Comment:
should we start using relay/client as terminology, or did we wanna stick with "router"?